src/hw java8/src/main/java/com/company/hw/books/Comics.java

```diff
@@ -0,0 +1,21 @@
+package com.company.hw.books;
+
+import com.company.hw.Library;
+import com.company.hw.interfaces.Artistic;
+
+public class Comics extends Library implements Artistic {
+
+    public Comics(String author, String name, int year, boolean avail){
+        super(author, name, year, avail);
+    }
+
+    @Override
+    public void typing(boolean type) {
```

Comment:
typing -> type (VERB) -> convention
boolean type -> the same -> should start with is OR has

Reply:
Fixed